grpc/server/CMakeLists.txt

```diff
@@ -1,74 +1,11 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
I remember we faced issue while testing with older cmake version(not sure how old though), that's why it was kept specific to 2.8.12
Hope you have tested with 2.8.0 so that anything above 2.8 will work.

Reply:
I can put it back to 2.8.12.